.gitignore

```diff
@@ -3,3 +3,4 @@
 .idea
 *version-info.json
 .DS_Store
+files/ecr-credential-provider/
```

Comment:
The `files` directory is mounted at `/etc/eks`, so we end up making this directory as part of the tests and it should up here. Cleaning up in the tests doesn't remove it, so just ignoring it.

Reply:
we should just `COPY` this into the container image instead of mounting it, that's what we do with all the other static resources. The only thing mounting in should be the test case being run, IMO

Reply:
Actually, looks like we already do that? https://github.com/awslabs/amazon-eks-ami/blob/master/test/Dockerfile#L10

Can you just remove this: https://github.com/awslabs/amazon-eks-ami/blob/master/test/test-harness.sh#L41

Reply:
Ya, let me see if that resolves the issue.